servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/GracefulConnectionClosureHandlingTest.java

```diff
@@ -146,26 +158,28 @@ class GracefulConnectionClosureHandlingTest {
     private final CountDownLatch serverSendResponse = new CountDownLatch(1);
     private final CountDownLatch serverSendResponsePayload = new CountDownLatch(1);
 
-    void setUp(HttpProtocol protocol, boolean initiateClosureFromClient,
+    private void setUp(HttpProtocol protocol, boolean secure, boolean initiateClosureFromClient,
                boolean useUds, boolean viaProxy) throws Exception {
+        setUp(protocol, secure, initiateClosureFromClient, useUds, viaProxy, false);
+    }
+
+    private void setUp(HttpProtocol protocol, boolean secure, boolean initiateClosureFromClient,
+               boolean useUds, boolean viaProxy, boolean withKeepAlive) throws Exception {
         this.protocol = protocol;
         this.initiateClosureFromClient = initiateClosureFromClient;
 
         if (useUds) {
-            Assumptions.assumeTrue(
-                    SERVER_CTX.ioExecutor().isUnixDomainSocketSupported(),
+            assumeTrue(SERVER_CTX.ioExecutor().isUnixDomainSocketSupported(),
                     "Server's IoExecutor does not support UnixDomainSocket");
-            Assumptions.assumeTrue(
-                    CLIENT_CTX.ioExecutor().isUnixDomainSocketSupported(),
+            assumeTrue(CLIENT_CTX.ioExecutor().isUnixDomainSocketSupported(),
                     "Client's IoExecutor does not support UnixDomainSocket");
             assumeFalse(viaProxy, "UDS cannot be used via proxy");
         }
         assumeFalse(protocol == HTTP_2 && viaProxy, "Proxy is not supported with HTTP/2");
```

Comment:
nit: could simplify to

```suggestion
if (viaProxy) {
  assumeFalse(!secure, "Proxy tunnel works only with secure connections");
  assumeFalse(protocol == HTTP_2, "Proxy is not supported with HTTP/2");
}

```